misc/scripts/svgs_2_pngs.py

```diff
@@ -10,8 +10,31 @@
 import rsvg
 import cairo
 
+
```

Comment:
You change this script in the first commit, then deprecate it. Wouldn't it be better to just remove it?